harper-core/src/spell/mod.rs

```diff
@@ -12,96 +10,126 @@ pub use self::merged_dictionary::MergedDictionary;
 mod dictionary;
 mod fst_dictionary;
 mod full_dictionary;
+mod hunspell;
 mod merged_dictionary;
 
-fn order_suggestions(matches: Vec<(&[char], u8, WordMetadata)>) -> Vec<&[char]> {
-    let mut found: Vec<(&[char], u8, WordMetadata)> = Vec::with_capacity(matches.len());
+/// Suggest a correct spelling for a given misspelled word.
+/// [`misspelled_word`] is assumed to be quite small (n < 100).
+/// [`max_edit_dist`] relates to an optimization that allows the search
+/// algorithm to prune large portions of the search.
+pub fn suggest_correct_spelling<'a>(
```

Comment:
Why are we overriding the logic from the individual dictionaries and just using the `FullDictionary` implementation?

Reply:
And not even the current one, this is the original implementation from `master` iirc?

Reply:
Spoke offline, this was unintentional.